Telerik.Examples.Mvc/Telerik.Examples.Mvc/Views/Grid/Grid_SignalR.cshtml

```diff
@@ -0,0 +1,75 @@
+﻿<script src="https://unpkg.com/@@aspnet/signalr@1.1.0/dist/browser/signalr.js"></script>
+<script>
+    var url = "/gridHub";
+
+    var hub = new signalR.HubConnectionBuilder()
+        .withUrl(url, {
+            transport: signalR.HttpTransportType.LongPolling
+        })
+        .build();
+
+    var hubStart = hub.start();
+
+
+    function onEdit() {
```

Comment:
I would recommend omitting the events since they are not utilized within the Grid's declaration. This will simplify the example.